### PR TITLE
Point CR source file to Venser's Journal

### PR DESCRIPTION
**Pull request by Villawhatever**
I have redirects set up on Venser's Journal that point at the most recent CR file, which I usually have downloaded and diffed within ~5 minutes of release. Ensures judgebot is getting up to date rules citations. Currently ~5mo out of date.

### CODE REVIEW

File: modules/rules/cr.js [hunk 1]
```diff
@@ -5,8 +5,7 @@ const utils = require("../../utils");
 const log = utils.getLogger('cr');
 const Discord = require('discord.js');
 
-// Using the current CR as the default, not sure if they actually stick around once new ones are published
-const CR_ADDRESS = process.env.CR_ADDRESS || "https://media.wizards.com/2020/downloads/MagicCompRules%2020200417.txt";
+const CR_ADDRESS = process.env.CR_ADDRESS || "cr.vensersjournal.com";
```
Comment (bra1n) [inline modules/rules/cr.js]: Would you mind adding the protocol (`http://`) to that URL?

**Comment by bra1n** [comment]
Awesome! I'll merge and deploy it when you've amended the URL. :-)

File: modules/rules/cr.js (hunk 1 above)
Comment (Villawhatever) [reply modules/rules/cr.js]: Done! 👍

**Comment by bra1n** [comment]
Awesome, tested and works. Finally I don't have to update the CR URL anymore. 😅

**Pull request merged**